create.ts

```diff
@@ -1,401 +1,104 @@
-      margin: 1rem auto 3rem;
-      font-size: clamp(2em, 10vw, 8em);
-      font-weight: 400;
-    }
-    
-    h1 span::before {
-      content: '@';
-      animation: blink 3s infinite;
-    }
-    
-    @keyframes blink {
-    
-      0%,
-      50%,
-      70%,
-      95% {
-        content: '@';
-      }
-    
-      65%,
-      90% {
-        content: '—';
-      }
-    }
-    
-    p {
-      max-width: 600px;
-      margin: 0 auto 1em;
-    }
-  `;
-}
-
-function helloUltraContent() {
-  // deno-fmt-ignore
-  return outdent`
-    export default function App() {
-      console.log("Hello world!");
-      return (
-        <html lang="en">
-          <head>
-            <meta charSet="utf-8" />
-            <title>Ultra</title>
-            <meta name="viewport" content="width=device-width, initial-scale=1" />
-            <link rel="shortcut icon" href="/favicon.ico" />
-            <link rel="stylesheet" href="/style.css" />
-          </head>
-          <body>
-            <main>
-              <h1>
-                <span></span>__<span></span>
-              </h1>
-              <p>
-                Welcome to{" "}
-                <strong>Ultra</strong>. This is a barebones starter for your web
-                app.
-              </p>
-              <p>
-                Take{" "}
-                <a
-                  href="https://ultrajs.dev/docs"
-                  target="_blank"
-                >
-                  this
-                </a>, you may need it where you are going. It will show you how to
-                customise your routing, data fetching, and styling with popular
-                libraries.
-              </p>
-            </main>
-          </body>
-        </html>
-      );
-    }
-  `;
-}
-
-function dialectExtension(dialect: Dialect, isJsx?: boolean) {
-  switch (dialect) {
-    case "js":
-      return isJsx ? "jsx" : "js";
-    case "ts":
-      return isJsx ? "tsx" : "ts";
-  }
-}
-
-if (import.meta.main) {
-  // deno-fmt-ignore
-  console.log(outdent`${brightBlue(`
-    ▄• ▄▌▄▄▌  ▄▄▄▄▄▄▄▄   ▄▄▄· 
-    █▪██▌██•  •██  ▀▄ █·▐█ ▀█ 
-    █▌▐█▌██▪   ▐█.▪▐▀▀▄ ▄█▀▀█ 
-    ▐█▄█▌▐█▌▐▌ ▐█▌·▐█•█▌▐█ ▪▐▌
-     ▀▀▀ .▀▀▀  ▀▀▀ .▀  ▀ ▀  ▀`)} \n
-    Let's get you setup with your new Ultra project.\n
-    But first, we need to ask just a few questions.\n
-  `);
-
-  let output = "";
-
-  while (output === "") {
-    output = await ask(
-      brightBlue("Where do you want to initialise your new project?"),
-    );
-  }
-
-  const typescript = await confirm(
-    brightBlue("Do you want to use TypeScript?"),
-  );
-  const dialect: Dialect = resolveDialect(typescript);
-
-  const context: TaskContext = {
+// IMPORTS
+import { Config, Libraries } from "./lib/create/common/config.ts";
+import { ask, confirm } from "./lib/create/common/ask.ts";
+import { createUltraApp } from "./lib/create/common/createUltraApp.ts";
+import { c, gradient } from "./lib/create/common/styling.ts";
+
+//INIT
+console.log(
```

Comment:
whats this for? seems odd

Reply:
Both functions are for colourizing text. `c(increment, text)` takes an increment (num) and adjusts the hue accordingly, it's for having each option be a different colour. `gradient()` is essentially the same function used in the Ultra CLI.